NEWS.md

```diff
@@ -1,3 +1,11 @@
+# hubVis 0.0.0.9101
```

Comment:
It looks like this should be 0.0.0.9201 to match the `DESCRIPTION`

Reply:
Ah yes, Good catch! Thank you!